tests/installation/addon_products_via_SCC_yast2.pm

```diff
@@ -53,11 +53,4 @@ sub run {
     assert_screen 'generic-desktop';
 }
 
-sub post_fail_hook {
```

Comment:
Why not to run it here?

Reply:
After rebasing my code I flipped the order and it fails in http://rivera-workstation.suse.cz/tests/2184#step/addon_products_via_SCC_yast2/186 so I'm going to switch to `script_output` there.

Reply:
VR added for this module as well.